src/main/groovy/org/jetbrains/intellij/Utils.groovy

```diff
@@ -332,4 +335,31 @@ class Utils {
         }
         return null
     }
+
+    @NotNull
+    static File downloadZipArtifact(@NotNull Project project, @NotNull String repoUrl, @NotNull String repoPattern,
+                                    @NotNull String dependency) {
+        debug(project, "Adding pseudo-Ivy repository to download $dependency - $repoUrl/$repoPattern")
+        def pseudoRepo = project.repositories.ivy {
+            url repoUrl
+            patternLayout {
+                artifact repoPattern
+            }
+            metadataSources {
+                artifact()
+            }
+        }
+        def projectDependency = project.dependencies.create("$dependency@zip")
+        def zipFile = null
+        try {
+            def configuration = project.configurations.detachedConfiguration(projectDependency)
+            zipFile = configuration.singleFile
+            debug(project, "Downloaded zip artifact: $zipFile")
+        } catch (Exception ignored) {
+
```

Comment:
shouldn't we log this?

Reply:
We return `null` in such a case so the consumer knows the method has failed to download the file (same situation in the `MavenPluginsRepository`). Do you think it's worth to log with debug level that it failed to download?

Reply:
Ah, got it. Probably `debug` log with the exception could be useful for future diagnostic